frontend/cypress/support/commands.ts

```diff
@@ -41,8 +41,98 @@
 //
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite("visit", (originalFn, url, options) => { ... })
-// @ts-expect-error there is no type definition for this package yet
-// eslint-disable-next-line no-restricted-imports
-import * as cypressImageDiffCommand from 'cypress-image-diff-js/dist/command';
 
-cypressImageDiffCommand();
+const performedActionsStack: any[] = [];
+
+export function dragToMap(
+    elementSelector: string,
+    offset?: { x: number; y: number }
+) {
+    cy.log(`dragging ${elementSelector} to the map`);
+    const mapSelector = '[data-cy=openLayersContainer]';
+
+    cy.get(elementSelector).first().trigger('mousedown');
+
+    return (
+        cy
+            .get(mapSelector)
+            /**
+             * We have to force the click here, as the img
+             * of the element being dragged is in front of the map
+             **/
+            .trigger('mousemove', { force: true, ...offset })
+            .click({ force: true })
+    );
+}
+
+export function storeState() {
+    return cy
+        .window()
+        .its('store')
+        .then((store: any) => store.source._value.application);
```

Comment:
```suggestion
        .then((store: any) => getStateSnapshot(store, selectApplication));
```

Or can't/shouldn't cypress import functions from the application?

We have  or something like this.

Reply:
I've tried this. The result is that ngrx errors because it can't find angular. But I've replaced it with a version that works and doesn't select these hacky values from the store.